css/index.css

```diff
@@ -112,6 +139,46 @@ p {
     justify-content: center;
 }
 
```

Comment:
Good job only overriding the CSS that needs to change in the child and letting the parent take care of the rest of the styling.